apps/api/src/controllers/auth.ts

```diff
@@ -75,23 +75,27 @@ export async function setCachedACUC(
 
 export async function getACUC(
   api_key: string,
-  cacheOnly = false
+  cacheOnly = false,
+  useCache = true
```

Comment:
Not caching ACUC is getting rid of like, 90% of the benefit of even having ACUC.

Reply:
This is only used once, to prevent race conditions on charging.